pkg/controller/csi_handler.go

```diff
@@ -192,8 +191,9 @@ func (h *csiHandler) prepareVANodeID(va *storage.VolumeAttachment, nodeID string
 	return clone, true
 }
 
-func (h *csiHandler) saveVA(va *storage.VolumeAttachment, patch []byte) (*storage.VolumeAttachment, error) {
-	newVA, err := h.client.StorageV1beta1().VolumeAttachments().Patch(va.Name, types.MergePatchType, patch)
+func (h *csiHandler) saveVA(va *storage.VolumeAttachment) (*storage.VolumeAttachment, error) {
+	// TODO: use patch to save us from VersionError
```

Comment:
can you link an issue (with a link to the original reverted Patch PR) for context

Reply:
added links to #158 and #157